cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
```

Comment:
How about
```
if iv, err := strconv.ParseBool(val) {
  return iv
}
```
as this will also parse TRUE, True, t, 1, etc.
https://golang.org/pkg/strconv/#ParseBool. But if we want to parse 1 as an integer we'd need to move this below.